lib/cocoapods_mangle/defines.rb

```diff
@@ -121,26 +121,44 @@ def self.prefix_selectors(prefix, selectors)
     def self.swift_symbol?(symbol)
       # Swift binaries have many symbols starting with $s_ that should be excluded
       # e.g. '0000000000000258 S _$s9ManglePod9SomeClassCMF'
-      symbol[/ _\$s/] ||
+      symbol[/\$s/] ||
         # Internal Swift symbols starting with __swift or ___swift such as should not be mangled
         # e.g. '00000000000050ac S ___swift_reflection_version' 
         symbol[/ __(_)?swift/] ||
-        # Swift symbols starting with _symbolic should be ignored
-        # e.g. '0000000000000248 S _symbolic _____ 9ManglePod9SomeClassC'
-        symbol[/ _symbolic/] ||
+        # Internal Swift symbols starting with Swift such as should not be mangled
+        # e.g. 'Swift51Override' 
+        symbol[/Swift/] ||
+        # Swift symbols starting with symbolic should be ignored
+        # e.g. '0000000000000248 S symbolic _____ 9ManglePod9SomeClassC'
+        symbol[/symbolic /] ||
         # Swift symbol references to Objective-C symbols should not be mangled
         # e.g. '00000000000108ca S _associated conformance So26SCNetworkReachabilityFlagsVs10SetAlgebraSCSQ'
         symbol[/associated conformance/] ||
         # _globalinit symbols should be skipped
         # e.g. 0000000000000000 T _globalinit_33_A313450CFC1FC3D0CBEF4411412DB9E8_func0
-        symbol[/ _globalinit/] ||
+        symbol[/ globalinit/] ||
```

Comment:
Unsure if this is correct as it doesn't match the comment description above of `_globalinit`? If the code is right, can we update the comment as well, or vice versa?